homeassistant/components/hue/services.py

```diff
@@ -0,0 +1,153 @@
+"""Handle HUE Service calls."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+from aiohue import HueBridgeV1, HueBridgeV2
+import voluptuous as vol
+
+from homeassistant.core import HomeAssistant, ServiceCall
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.service import verify_domain_control
+
+from .bridge import HueBridge
+from .const import (
+    ATTR_DYNAMIC,
+    ATTR_GROUP_NAME,
+    ATTR_SCENE_NAME,
+    ATTR_TRANSITION,
+    DOMAIN,
+    SERVICE_HUE_ACTIVATE_SCENE,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+async def async_register_services(hass: HomeAssistant) -> None:
+    """Register services for Hue integration."""
+
+    async def hue_activate_scene(call: ServiceCall, skip_reload=True):
```

Comment:
We can deprecate this one now that we offer scene entities?

Reply:
That would make it a breaking change... Leaving it in (it's now compatible with both v1 and v2) doesn't break existing automations etc. What do you think ? Only add the service for v1 bridges and make this a breaking change ?

Reply:
Deprecating would mean we print a warning to use the scene entities. We can do that for just v2 maybe?

Reply:
makes sense! I'll ad a warning log message in there

Reply:
warning message added, will be printed only when making the call on a v2 bridge